nss-defective-foxes/nutshell
# Task list

## Link to Ticket
Closes #4

## Description of Proposed Changes

- adds task list linked to user

## Steps to Test

upon login-
-displays task adding feature 
-add task
*Note - doing the editing steps too quickly WILL break the feature but a refresh will fix it*
-use edit button attached to new task to open an editing field filled in with the information attached to the current task
change
save
return the same number of tasks as previously present
-delete button
removes from DB and no longer displays


```sh
git fetch --all
git checkout taskList
```



## Impacted Areas in Application

List general components of the application that this PR will affect:

*Index.html div added
*taskList.js page added
*load js page requires and calls taskList
*

## Mentions @username

Tag users that need to review this code

## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
